### PR TITLE
[12.0][ADD] Add Ticket Report

### PR DESCRIPTION
**Pull request by marcelsavegnago**
Add Ticket Report.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#14directories

### CODE REVIEW

File: helpdesk_mgmt/views/report_helpdesk_ticket.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+                            <p class="m-0" t-field="doc.team_id" />
+                        </div>
+
+                        <div t-if="doc.user_id" class="col-auto mw-100 mb-2">
+                            <strong>Assigned user:</strong>
+                            <p class="m-0" t-field="doc.user_id" />
+                        </div>
+
+                        <div t-if="doc.create_date" class="col-auto mw-100 mb-2">
+                            <strong>Create on:</strong>
```
Comment (dalonsod) [inline helpdesk_mgmt/views/report_helpdesk_ticket.xml]: Typo:
```suggestion
                            <strong>Created on:</strong>
```

File: helpdesk_mgmt/views/report_helpdesk_ticket.xml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                            <strong>Tag:</strong>
+                            <p class="m-0" t-field="doc.tag_ids" />
+                        </div>
+                    </div>
+
+                    <div class="row mt32 mb32" id="description">
+                        <div t-if="doc.description" class="col-auto mw-100 mb-2">
+                            <h3>Description:</h3>
+                            <p class="m-0" t-field="doc.description" />
+                        </div>
```
Comment (dalonsod) [inline helpdesk_mgmt/views/report_helpdesk_ticket.xml]: I think there's something wrong here with the description width. If you print a report with a long description, you can see that only two columns are filled

**Review by dalonsod — CHANGES_REQUESTED** [review]
Functional and code review

File: helpdesk_mgmt/views/report_helpdesk_ticket.xml (hunk 2 above)
Comment (marcelsavegnago) [reply helpdesk_mgmt/views/report_helpdesk_ticket.xml]: I didn't notice the problem. Take a look at the file.
image

Comment (dalonsod) [reply helpdesk_mgmt/views/report_helpdesk_ticket.xml]: Well, I've tested on runbot, this is the result with a demo ticket data (rendered by Firefox):

imagen

The related demo data, unchanged:

imagen

It should be expected to fill only one row within the report, shouldn't it be?

Comment (marcelsavegnago) [reply helpdesk_mgmt/views/report_helpdesk_ticket.xml]: Now it looks right

Comment (marcelsavegnago) [reply helpdesk_mgmt/views/report_helpdesk_ticket.xml]: @dalonsod Try again please.

**Review by dalonsod — APPROVED** [review]
👍 for me. Only squash commits pending.

**Review by dreispt — APPROVED** [review]

**Pull request merged**